.buildkite/pipeline.build_core.yml

```diff
@@ -1,7 +1,6 @@
 #ci:group=:book: core tests (civ2)
 
 - label: ":python: (small & client)"
-  conditions: ["RAY_CI_PYTHON_AFFECTED"]
```

Comment:
why is this line removed?

Reply:
i was removing it for ci to trigger the job for testing, added back